frontend/packages/dev-console/src/components/import/section/ResourceSection.tsx

```diff
@@ -64,9 +64,11 @@ const ResourceSection: React.FC<ResourceSectionProps> = ({ flags }) => {
       label: (
         <div>
           Knative Service
-          <span className="odc-resource-section__badge">
-            {getBadgeFromType(KnativeServingModel.badge)}
-          </span>
+          <div className="odc-resource-section__badge-wrapper">
+            <span className="odc-resource-section__inline-badge">
```

Comment:
Perhaps make both of these spans, since the styles:
```
  &__badge-wrapper {
    display: inline;
```
Are just getting rid of the `div`s block nature anyways. And `div`s for the most part are nothing but block containers.